config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/EntityUtils.java

```diff
@@ -109,5 +123,10 @@ public FileType getFileType() {
         public String getEnvironmentType() {
             return environmentType;
         }
+
+        public boolean isBundleGenerationSupported() {
```

Comment:
Why can't you silently compute this value inside?
You have entityClass property to do so.

Reply:
done